Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG STELLAR_CORE_VERSION=18.5.0-877.d387c6a71.focal
+ARG STELLAR_CORE_VERSION=19.0.0-907.9d0704eb4.focal
```

Comment:
Could we make the change only to testing in this PR? Horizon has been updated in testing now.